x-pack/plugins/security_solution/common/api/entity_analytics/risk_engine/engine_schedule_now_route.schema.yaml

```diff
@@ -0,0 +1,62 @@
+openapi: 3.0.0
+
+info:
+  version: '1'
+  title: Risk Scoring API
+  description: These APIs allow the consumer to manage Entity Risk Scores within Entity Analytics.
+
+servers:
+  - url: 'http://{kibana_host}:{port}'
+    variables:
+      kibana_host:
+        default: localhost
+      port:
+        default: '5601'
+
+paths:
+  /internal/risk_score/engine/schedule_now:
+    post:
+      x-labels: [ess, serverless]
+      x-internal: true
+      operationId: ScheduleRiskEngineNow
+      summary: Schedule the risk engine to run as soon as possible
+      requestBody:
+        content:
+          application/json: {}
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/RiskEngineScheduleNowResponse'
+        '400':
```

Comment:
In what situation is the task manager unavailable? 

I would take 400 as there is something wrong with the users request format/syntax. Is there a better status code we can use here? is this a 503?

Reply:
This is a copy and paste 😞 
 
All risk engine routes have the same 400 response when the task manager is unavailable: https://github.com/elastic/kibana/blob/dac41c678c83c3da3adf55f69ab43bfb8e532927/x-pack/plugins/security_solution/common/api/entity_analytics/risk_engine/engine_enable_route.schema.yaml#L27

https://github.com/elastic/kibana/blob/bc79b994d1f26c13f09cfe538476b1ad59d51efe/x-pack/plugins/security_solution/server/lib/entity_analytics/risk_engine/routes/disable.ts#L70

What do you think about updating this status code for all APIs in a following PR?

Reply:
Ah interesting :D Would you mind creating a little tech debt issue and adding it to the prioritised backlog?